CHANGELOG.md

```diff
@@ -8,6 +8,27 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 
 ## [Unreleased]
 
+## [1.1.1] - 2023-06-26
+
+[Full changelog](https://github.com/openfga/openfga/compare/v1.1.0...v1.1.1)
+
+### Added
+* Official Homebrew installation instructions (#781) - thanks @chenrui333
+* The `--verbose` flag has been added to the `openfga migrate` command (#776)
+* The `openfga validate-models` CLI command has been introduced to validate all models across all stores (#817)
+
+
```

Comment:
Can you remove the extra new line here? (not a blocker)

Reply:
Removed it 👍